src/GitHub.InlineReviews/Services/PullRequestStatusBarManager.cs

```diff
@@ -84,7 +84,8 @@ void RefreshActiveRepository(ILocalRepositoryModel repository)
         void RefreshCurrentSession(ILocalRepositoryModel repository, IPullRequestSession session)
         {
             var cloneUrl = repository?.CloneUrl;
-            if (cloneUrl != null)
+            var host = cloneUrl?.Host;
+            if (host != null && host.Equals("github.com", StringComparison.OrdinalIgnoreCase))
```

Comment:
This could be written more succinctly as `cloneUrl?.Host.Equals("github.com", StringComparison.OrdinalIgnoreCase) == true` I think.